src/pages/api/assistant/trip.ts

```diff
@@ -0,0 +1,59 @@
+import { parseFilterQuery } from '@atb/components/transport-mode-filter/utils';
+import { errorResultAsJson, tryResult } from '@atb/modules/api-server';
+import { TripApiReturnType } from '@atb/page-modules/assistant/client/journey-planner';
+import { handlerWithAssistantClient } from '@atb/page-modules/assistant/server';
+import { parseTripQuery } from '@atb/page-modules/assistant/utils';
```

Comment:
Same as above. But maybe here it could be exported by `@atb/page-modules/assistant` and that will be ok